reference/wishlists.v3.yml

```diff
@@ -226,7 +239,7 @@ paths:
         required: true
   '/wishlists/{wishlist_id}':
     get:
-      description: Returns a single Wishlist.
+      description: Returns a single wishlist.
       summary: Get a Wishlist
```

Comment:
```suggestion
      summary: Get a wishlist
```

Reply:
Changing the summary field will change the way the name of the endpoint appears in the left pane of API Reference.